[DOCUMENTATION] Remove direct references to team members (#4226)
ClanGenOfficial/clangen: docs/dev/writing/index.md

        - For singular words that end in S, add an apostrophe to make them possessive. For instance, "the crocus' petals". For a possessive plural of a word that ends in S, add "es" to the end, then apostrophe. Random example sentence: "The actresses' rooms were located across the hall."
        - Basically, anytime there's a possessive, you want an apostrophe in there somewhere. If the word ends in S, plural or singular, the apostrophe goes at the end. If the singular form of the word ends in S and it's a plural possessive, add "es" at the end, then apostrophe.  The exception to this is “it’s’ and “its”, where “its” is the possessive form and has no apostrophe.
- All ClanGen game text should follow normal grammar rules for capitalizing the first letter of a sentence, and trying to avoid spelling or grammar typos. We all make typos, don't worry! But this is one of the ways beta testing your new content can help you!

> - If you are struggling with remembering or understanding a grammar rule, even one that isn't mentioned in the above list, feel free to message in the writing contributor thread or in other offical channels of the discord server.

Reviewer: ```suggestion
- If you are struggling with remembering or understanding a grammar rule, even one that isn't mentioned in the above list, feel free to message in the writing contributor thread or in other official channels of the Discord server.
```

- HTML tags for bolding and italicizing are usable. These tags are:
    - `<i>` and `</i>` for italicizing
    - `<b>` and `</b>` for bolding

> - Keeping a change log: as a writer, people are going to be super excited to experience what you write! Make sure to keep a rough record of what that might be, i.e A writer has no idea how many fox patrols they have added total, but does know how many species are added are in what biomes, so that when they made the PR to merge them into the game they could mention their existence in the changelog for that PR.

Reviewer: This is an anecdote from a specific writer's experience, so perhaps it should be noted that "a given writer" so that it's clear it's not just a generic mass noun "writer"

Author: That is true. I was editing that as if it was strictly an example of the point made, assuming that was the original reason it was used, rather than it being being anecdotal. If it would read better being specified more directly as an anecdote, that change can be made, but I'm not certain it's neccessary?

> - Keeping a change log: as a writer, people are going to be super excited to experience what you write! Make sure to keep a rough record of what that might be, i.e A writer has no idea how many fox patrols they have added total, but does know how many species are added are in what biomes, so that when they made the PR to merge them into the game they could mention their existence in the changelog for that PR.

Reviewer: ```suggestion
- Keeping a change log: as a writer, people are going to be super excited to experience what you write! Make sure to keep a rough record of what that might be. For example, a writer may have no idea how many fox patrols they are adding total. However, they do know how many species they have added and in what biomes. So, when they make the PR to merge them into the game, they can mention their existence in the changelog for that PR.
```
